diagnostic.md

```diff
@@ -40,15 +48,15 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is an application that looks at HTML like tree structures and represents objects as nodes... it's an object that the browser creates for us so we can see a visual representation of what we're doing. It helps us 'change' HTML with JavaScript without actually changing the HTML.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.